src/components/Card/Card.tsx

```diff
@@ -28,17 +28,14 @@ const Card: React.StatelessComponent<CardProps> = (props) => {
     return (
         <SmCard>
             <CardContent>
-                <AssigneesAvatar assignees={props.card.assignees} />
+                <AssigneesAvatar assignees={props.card.members} />
                 <CardMeta>
-                    <LabelsList labels={props.card.labels} maxToDisplay={2} />
+                    <LabelsList labels={props.card.tags} maxToDisplay={2} />
                 </CardMeta>
                 <CardDescription>
-                    {props.card.title}
+                    {props.card.name}
                 </CardDescription>
             </CardContent>
-            {props.card.percent !== 0 &&
```

Comment:
Why do you remove this ?

Reply:
How the percent it is calculated ? I removed it because for now I don't handle it in my tests, and Card don't have a percent attribute. Do I have to add a new attribute percent to Card ?

Reply:
For now, add a percent attribute, we will see later how to handle this.